### ISSUE TITLE
Check the Livebook

### ISSUE THREAD

**Issue by ndrean** [dwyl/learn-elixir-with-livebook#1]
Please review it. The Livebook code can be modified as well.
Thks!

**Comment by nelsonic** [comment]
@ndrean looking really good so far. 👌 
However I'm quite keen not fragment our `Elixir` learning into a **`new`** repo ... 🤔 
Prefer to integrate this directly into the main `learn-elixir` repo 
and just include the "**Run in Livebook**" button there. 🚀 
i.e. split the "Learn Elixir" section: https://github.com/dwyl/learn-elixir#learn-elixir
into a new [`.livemd`](https://github.com/dwyl/learn-elixir-with-livebook/blob/main/learn-elixir-on-livebook.livemd) file.

Thoughts? 💭

**Comment by ndrean** [comment]
You are right, it is not another repo, should be a paragraph. However, be aware it is not 100% the same,  mainly because you have to choose between Ipresenting things with IEX or Livebook. I also have to recheck if I did not invent things during the lengthy copy-paste formatting process :)  And you see it is hacky  to run a Livebook hosted on Fly.io, or it is me?  Did you try both ways?

**Comment by nelsonic** [comment]
I think hosting it on Fly.io is perfectly acceptable. 💭
